### PR TITLE
Catch everything in command route

### LINKED ISSUE #21: 404 is thrown if URL with scheme is defined in URL
Go to https://dnsrecords.io/https://spatie.be. This'll throw a 404 because there's a `/` in the URL after the domain name. This can be fixed by adding `->where('command', '\S+');` after [this line](https://github.com/spatie/dnsrecords.io/blob/master/routes/web.php#L4).

### PR DESCRIPTION
**Pull request by svenluijten**
Fixes #21

### CODE REVIEW

**Comment by svenluijten** [comment]
"Proof" that this works: 
chrome_2017-10-30_15-07-23

File: routes/web.php [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <?php
 
 Route::get('/', 'HomeController@index')->name('home');
-Route::match(['get', 'post'], '/{command}', 'HomeController@submit')->middleware('logRequest');
+Route::match(['get', 'post'], '/{command}', 'HomeController@submit')->middleware('logRequest')->where('command', '\S+');
```
Comment (freekmurze) [inline routes/web.php]: Could you change the regex to `.*`?

Comment (svenluijten) [reply routes/web.php]: done

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**